content/app-dev/debugging/host.en.md

```diff
@@ -15,7 +15,7 @@ The wasmCloud host logs include the following:
 1. Logs from actors that utilize the `wasmcloud:builtin:logging` capability
 
 ## Finding the logs (release tarball)
-If you followed the [installation](../../../overview/installation/) guide then you likely started the host by running a command like `./bin/wasmcloud_host start` or `./bin/wasmcloud_host foreground`. In the same directory where you unpacked the tarball you'll be able to find the logs in `./var/log/erlang.log.X`, where `X` starts at `1` and will increment monotonically as the log file grows too large.
+If you followed the [installation](../../../overview/installation/) guide then you likely started the host by running a command like `./bin/wasmcloud_host start` or `./bin/wasmcloud_host daemon`. In the same directory where you unpacked the tarball you'll be able to find the logs in `./tmp/log/erlang.log.X`, where `X` starts at `1` and will increment monotonically as the log file grows too large.
```

Comment:
Is this directory always the same even on windows?

Reply:
Yep!